src/NetEscapades.AspNetCore.SecurityHeaders/HeaderPolicyCollectionExtensions.cs

```diff
@@ -80,6 +83,12 @@ public static HeaderPolicyCollection AddDefaultApiSecurityHeaders(this HeaderPol
         // The following are generally not applicable, but still worth applying for safety
         policies.AddReferrerPolicyNoReferrer();
         policies.AddPermissionsPolicyWithDefaultSecureDirectives();
+
+        policies.AddReferrerPolicyStrictOriginWhenCrossOrigin();
```

Comment:
I think we can remove this one, as we already have the stricter `AddReferrerPolicyNoReferrer()` above?